doc/cookbook/source/examples/multiclass_classifier/cartree.rst

```diff
@@ -0,0 +1,47 @@
+==================================
+Classification And Regression Tree
+==================================
+
+Decision tree learning uses a decision tree as a predictive model which maps observations about an item to conclusions about the item's target value.
+
+Decision trees are mostly used as the following two types:
```

Comment:
Or even better: There are decision trees for both classification (integer-valued) and regression (real-valued).